website/content/docs/gridsome.md

```diff
@@ -0,0 +1,164 @@
+
+// Using Yarn
+yarn global add @gridsome/cli
+
+// Using NPM
+npm install --global @gridsome/cli
+```
+
+## Create a new Gridsome website
+
+``` javascript
+// To create a new project run
+gridsome create gridsome-netlify-blog
+
+// Then open folder
+cd gridsome-netlify-blog
+
+// To start local dev server at `http://localhost:8080
+gridsome develop
+```
+
+### Install Netlify CMS the required dependencies to your project
+
+``` javascript
+
+// Using Yarn
+yarn add netlify-cms gridsome-plugin-netlify-cms @gridsome/source-filesystem @gridsome/transformer-remark
+
+// Using NPM
+npm add netlify-cms gridsome-plugin-netlify-cms @gridsome/source-filesystem @gridsome/transformer-remark
+```
+
+Now that the plugins are installed, it's now time to setup the right configuration. Open the `gridsome.config.js` file and make sure it looks like this:
+
+```js
+module.exports = {
+  siteName: 'Gridsome',
+  transformers: {
+    remark: {
+      externalLinksTarget: '_blank',
+      externalLinksRel: ['nofollow', 'noopener', 'noreferrer'],
+      anchorClassName: 'icon icon-link',
+      plugins: [
+        // ...global plugins
+      ]
+    }
+  },
+
+  plugins: [
+    {
+      use: '@gridsome/source-filesystem',
+      options: {
+        path: 'posts/**/*.md',
+        typeName: 'Post',
+        remark: {
+          plugins: [
+            // ...local plugins
+          ]
+        }
+      }
+    },
+    {
+      use: `gridsome-plugin-netlify-cms`,
+      options: {
+        publicPath: `/admin`
+      }
+    },
+  ]
+}
+```
+
+Please read [gridsome-plugin-netlify-cms](https://gridsome.org/plugins/gridsome-plugin-netlify-cms), [transformer-remark](https://gridsome.org/plugins/@gridsome/transformer-remark) for more information about the configurations.
+
+## Netlify CMS setup
+
+1. Create an `admin` directory inside the `src`
+2. Create an `uploads` directory inside the root of your project
+3. Add `index.html`, `index.js` and a `config.yml` file to your `admin` directory
+
+Your `index.html` should look like this:
+
+```html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Netlify CMS</title>
+</head>
+<body>
+  <script src="index.js" type="module"></script>
+</body>
+</html>
+```
+
+Your `index.js` should look like this:
+
+```js
+import CMS from "netlify-cms"
+```
+
+Your `config.yml` for GitHub should look like this:
+
+```yml
+backend:
+  name: github
+  repo: your_name/repo_name
+
+media_folder: "static/uploads"
+public_folder: "/uploads"
+
+collections:
+  - name: "posts"
+    label: "Posts"
+    folder: "posts"
+    create: true
+    slug: "{{slug}}"
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Excerpt", name: "excerpt", widget: "string"}
+      - {label: "Publish Date", name: "date", widget: "datetime"}
+      - {label: "Body", name: "body", widget: "markdown"}
+```
+
+## Add your repo to Netlify
```

Comment:
Should we add the missing `Push to GitHub` part like the other guides?
https://www.netlifycms.org/docs/gatsby/#push-to-github